resources/com/soywiz/vitaorganizer/Texts_de.properties

```diff
@@ -1,73 +1,73 @@
 PERMISSIONS_UNSAFE=\!UNSICHER\!
-PSF_VIEWER_TITLE=PSF Betrachter \: %id% \: %title%
-SELECT_PSVITA_VPK_FOLDER=PS VITA VPK Ordner ausw\u00E4hlen
+PSF_VIEWER_TITLE=PSF Betrachter: %id% \: %title%
+SELECT_PSVITA_VPK_FOLDER=PS Vita VPK Ordner ausw\u00E4hlen
 SEND_DATA_TO_VITA_ACTION=Sende Daten zur PS Vita
-SEND_FULL_APP_TO_VITA_ACTION=Installiere Applikation in nur einem Schritt (erfordert VitaShell >\= 0.9.5)
-SEND_PROMOTING_VPK_TO_VITA_ACTION=Sende Vorab-Installations VPK zur PS Vita
+SEND_FULL_APP_TO_VITA_ACTION=Installiere APP in nur einem Schritt (erfordert VitaShell >\= 0.9.5)
+SEND_PROMOTING_VPK_TO_VITA_ACTION=Sende Promoting-VPK zur PS Vita
 STEP_ANALYZING_FILES=Analysiere Dateien in %folder%...
 STEP_ANALYZING_ITEM=Analysiere %name% %current%/%total%...
 STEP_CHECKING_EBOOT_PERMISSIONS=\u00DCberprüfe eboot Berechtigungen...
 STEP_DONE=Fertig
-STEP_GENERATED_SMALL_VPK_FOR_PROMOTING=Vorab-Installations VPK wurde generiert
-STEP_GENERATING_SMALL_VPK_FOR_PROMOTING=Generiere Vorab-Installations VPK...
+STEP_GENERATED_SMALL_VPK_FOR_PROMOTING=Promoting-VPK wurde generiert
+STEP_GENERATING_SMALL_VPK_FOR_PROMOTING=Generiere Promoting-VPK...
 STEP_STARTED=Gestartet
-STEP_UPLOADING_VPK_FOR_PROMOTING=\u00DCbertrage Vorab-Installations VPK %current%/%total%...
+STEP_UPLOADING_VPK_FOR_PROMOTING=\u00DCbertrage Promoting-VPK %current%/%total%...
 UNKNOWN_VERSION=Unbekannt
 LABEL_FILTER=Filter:
 YOU_HAVE_LASTEST_VERSION=Du besitzt die letzte verf\u00FCgbare Version: %version%
-DUMPER_VERSION=Dumper : %version%
-COMPRESSION_LEVEL=Compression Level : %level%
-CONNECT_TO_PSVITA=Connect to PsVita...
-DISCONNECT_FROM_IP=Disconnect from %ip%
-DISCONNECTED=Disconnected
-DISCONNECTING=Disconnecting...
-CONNECTING=Connecting...
-CONNECTED=Connected
-PROCESSING_GAME=Processing game %current%/%total% %id%...
-KEY_COLUMN=Key
-VALUE_COLUMN=Value
-ACTIONS_TITLE=Actions
-PROMOTING_VPK=Promoting VPK (this could take a while)...
-GAME_SENT_SUCCESSFULLY=Game %id% sent successfully
-STEP_REPACKING_VPK=Repacking vpk...
-STEP_REPACKING_ENTRY=Repacking %current%/%total% :: %currentSize%/%totalSize%
-STEP_SENDING_GAME=Sending game %id%
-NEW_VERSION_AVAILABLE=There is a new version: %lastVersion%\nYou have: %currentVersion%\nWant to download last version?
-SENT_GAME_DATA=Sent game data %id%
-WARNING_EX=WARNING!
-UNSAFE_NOT_ACCEPTED=Not accepted installing game with extended permissions
-WARNING_UNSAFE=Game %id% requires extended permissions.\nAre you sure you want to install it. It could damage your device?
-VITASHELL_INSTALL=Now use VitaShell to install\n%vpkPath%\n\nAfter that active ftp again and use this program to Send Data to PSVita
-METHOD1_INFO=Old method: 2 steps - Compatible with all ftp and VitaShell 0.9.1 and lower
-METHOD2_INFO=New method: 1 single step - Requires VitaShell 0.9.5 and up
-ABOUT_TITLE=About
-ABOUT_PROGRAMMING=Programming:
-ABOUT_TRANSLATIONS=Translations:
+DUMPER_VERSION=Dumper: %version%
+COMPRESSION_LEVEL=Kompressionsstufe: %level%
```

Comment:
I will change it in the new version, for that I will change it from %level% to method